include/tallies/position_filter.hpp

```diff
@@ -32,6 +32,10 @@ class PositionFilter {
 
   virtual StaticVector3 get_shape() const = 0;
 
+  // method to get the volume and inverse of the volume
+  virtual double dV(StaticVector3 index) const = 0;
+  virtual double inv_dV(StaticVector3 index) const = 0;
```

Comment:
```suggestion
  virtual double dV(const StaticVector3& index) const = 0;
  virtual double inv_dV(const StaticVector3& index) const = 0;
```

Reply:
As discussed last time, I have removed the functions to get the volumes and inverse of it. Therefore, we will not be calling these functions anywhere else.